packages/upgrade/src/bin/cli.ts

```
@@ -0,0 +1,238 @@
+/* eslint-disable @typescript-eslint/unbound-method */
+import path from 'node:path';
+import { Command as CLICommand, Options, Prompt } from '@effect/cli';
+import { Command } from '@effect/platform';
+import { NodeContext, NodeRuntime } from '@effect/platform-node';
+import {
+  Array,
```

Biome (1.9.4)

[error] Do not shadow the global "Array" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)